imageio/imageio-pict/src/main/java/com/twelvemonkeys/imageio/plugins/pict/PICTImageReader.java

```diff
@@ -1877,10 +1880,10 @@ else if (rowBytes > 250) {
             context.copyBits(img, srcRect, dstRect, transferMode, region);
         }
 
-        // Line break at the end
-        if (DEBUG) {
-            System.out.println();
-        }
+//      // Line break at the end
+//      if (DEBUG) {
+//          System.out.println();
+//      }
```

Comment:
I think we can just remove these line, no need to comment out. We have version control. 😀